docs/manual/getting_started.rst

```diff
@@ -173,21 +190,23 @@ This step will copies everything to:
 At this point, you are ready to use Eclipse Cyclone DDS in your
 projects.
 
-**Note:** The default build type is a release build with debug
-information included (RelWithDebInfo). This is a convenient type of
-build to use from applications because of a good mix between performance
-and still being able to debug things. If you'd rather have a Debug or
-pure Release build, set ``CMAKE_BUILD_TYPE`` accordingly.
+**Note:** The default build type is a release build that includes debugging 
+information (RelWithDebInfo). 
+
+This is a convenient type of build to use from applications because 
+of a good mix between performance and still being able to debug things. 
+If you'd rather have a Debug or pure Release build, 
+set ``CMAKE_BUILD_TYPE`` accordingly.
```

Comment:
I know this is just a format change, but the content reads really weirdly. Especially lines 196-197.